### DISCUSSION TITLE
How to create backups

### DISCUSSION THREAD

**Discussion by mapa17** [Leantime/leantime#523] [Q&A]
Hello, 
I am thinking about using leantime in a self-hosted setting (with the docker images), and was wondering what my options there are for creating backups?
best,
Manuel

**Comment by marcelfolaron** [comment]
The simplest way to run updates right now is to:

1. Take a mysql dump of the db
2. Take a back up of userfiles/ and public/userfiles/

To restore an update you can just use a fresh leantime install and update the db info
